modal/__init__.py

```diff
@@ -79,4 +80,5 @@
     "web_endpoint",
     "web_server",
     "wsgi_app",
+    "LocalFileFilter",
```

Comment:
I'm not understanding why we need this new public object.

Reply:
Removed!